newIDE/app/src/Profile/ProfileDetails.js

```diff
@@ -40,25 +44,43 @@ const ProfileDetails = ({
   onEditProfile,
   badges,
 }: Props) => {
+  const isDev = process.env.NODE_ENV === 'development';
   return profile ? (
```

Comment:
You should have something for this in the Window.js file :)

Reply:
Actually we don't need it here anymore since I moved this check in the Utils function's file. Should I still use Window.isDev() in the utils file ? (If it is the case, I saw the node_env check in ApiConfig, should it be changed too ?)

Reply:
I believe we can change the node_env usage in the ApiConfig too. 
Change it and ensure api calls are going to the right endpoints locally?